### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by Mahadevinc**
updated notes for sem-5,6,7 and 8
updated my profile in contributors

### CODE REVIEW

File: src/notes_details.json [hunk 1]
```diff
@@ -320,5 +320,862 @@
...
+                    {
+                        "id": "mod5",
+                        "title": "Module 5",
+                        "link": "https://studentsfocus.com/notes/anna_university/2017/CSE/6th%20Sem/Units%20Notes/CS8601%20Mobile%20Computing/Unit%205/104625.pdf"
+                    }
+                ]
+            },
+            {
+                "id": 3,
+                "name": "Storage area Networks",
```
Comment (akramkazmi71) [inline src/notes_details.json:1108]: I think this is in 7th sem, can you check that once.

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Comment by akramkazmi71** [comment]
Also, if possible can you add 1st and 2nd sem notes as well?

File: src/notes_details.json (hunk 1 above)
Comment (Mahadevinc) [reply src/notes_details.json:1108]: I just followed the subjects as in 18 scheme syllabus. they have mentioned 8th sem in subject code and mentioned as semester 7. I just followed with the subject code


Screenshot (673)

**Comment by Mahadevinc** [comment]
> Also, if possible can you add 1st and 2nd sem notes as well?

File: src/notes_details.json (hunk 1 above)
Comment (akramkazmi71) [reply src/notes_details.json:1108]: Add this in 7th sem.

**Comment by Mahadevinc** [comment]
I have uploaded 1st and 2nd sem notes. and moved SAN notes to 7th sem

File: src/notes_details.json [hunk 2]
```diff
@@ -1,324 +1,1547 @@
...
+                        "title": "Complete Notes",
+                        "link": "https://mrcet.com/downloads/hs/Engineering%20Graphics%20Manual%20final.pdf" 
+                    }
+                    
+                ]
+            }
+        ]
+    },
+    {
+        "id": 1,
```
Comment (akramkazmi71) [inline src/notes_details.json:145]: id's should always be unique.

**Review by akramkazmi71 — CHANGES_REQUESTED** [review]

**Review by akramkazmi71 — APPROVED** [review]

**Pull request merged**